index.html

```diff
@@ -36,6 +36,168 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
     </ul>
     </div>
 </section>
-  
+<section class="work-section">
+    <div class="tonic-container">
+        <div><img src="images/tonic-portfolio.png" alt="tonic"></div>
+        <div class="tonic-heading">Tonic</div>
+        <div class="canopy-container">
```

Comment:
- [ ] The project title ("Tonic") should use a header tag `hx` (Ex: `<h2>`, `<h3>` etc.) :+1: